### ISSUE TITLE
Number of digits in examples with coordinate formats

### ISSUE THREAD

**Issue by pruzhinskaya** [astrolabsoftware/fink-science-portal#70]
Once the coordinates are transformed to hours, m, s for alpha and degrees, arcmin, arcsec for delta,  the number of  digits after the decimal point can not be the same: one more significant digit for alpha is needed. 
For example, 13h 09m 47.706s -23d 23' 01.79''

**Comment by JulienPeloton** [comment]
Thanks @pruzhinskaya for reporting it. I do not think I get your point - so let me explain what I have in the example:

```python
from astropy.coordinates import SkyCoord

ra_deg = 271.3914265
dec_deg = 45.2545134

coord = SkyCoord(ra_deg, dec_deg, unit='deg')

print(coord.ra.hms, coord.dec.dms)
# hms_tuple(h=18.0, m=5.0, s=33.942360000021665), dms_tuple(d=45.0, m=15.0, s=16.2482400000016)
```

which I rounded to the 4th digit (`18h05m33.9424s` and `45d15m16.2482s`) in the example. What is wrong according to you then?

**Comment by pruzhinskaya** [comment]
When alpha and delta are in degrees (the same units), they have the same accuracy (not true exactly but it is not related to this issue). When you transform alpha to hours min sec, the number of significant digits cannot be the same for both, since the units become different now. In your example, it should be 18h05m33.9424s and 45d15m16.248s. 
1 arcsec is 15 * second.

**Comment by JulienPeloton** [comment]
Oh I see - thanks for pointing this out. I will correct the documentation then. 

Fortunately, in the code the transformation is done programmatically so there is no such rounding:

https://github.com/astrolabsoftware/fink-science-portal/blob/74b7ca0f45d23cd86de8335db8ffaa1dbe8adb6c/apps/explorer.py#L613-L633

**Comment by pruzhinskaya** [comment]
Thanks, Julien. It is not a super important issue and should not affect anything, but for the beauty. And it is usually like that in the astronomical databases and catalogs where the coordinates are given in h m s for RA and deg ' '' for Dec.

<img width="766" alt="изображение" src="https://user-images.githubusercontent.com/16917953/101737183-965c2b80-3ad5-11eb-852a-f71807301452.png">

**Comment by JulienPeloton** [comment]
Fixed in #72. Thanks @pruzhinskaya !
